main.py

```diff
@@ -148,5 +149,76 @@ async def keen(context):
     await context.send(result)
 
 
-bot.run(TOKEN)
+@bot.command(name='daily', help='Will give the daily leaderboard for specified day')
+async def daily(context, day : str = str((datetime.datetime.today() - datetime.timedelta(hours=5)).day)):
+    # The default day is whatever day's challenge has just come out.  So at 4.59AM UTC will still show previous day's leaderboard
```

Comment:
There's two spaces at the start of the second sentence

Reply:
Sorry normally do 2 spaces!